lnwire/update_fail_htlc.go

```diff
@@ -19,28 +20,28 @@ const (
 	// InsufficientCapacity indicates that a payment failed due to a link
 	// in the ultimate route not having enough satoshi flow to successfully
 	// carry the payment.
-	InsufficientCapacity FailCode = 0
+	InsufficientCapacity FailCode = 1
```

Comment:
What's the rationale behind this change? If there's none, it should be reverted.